src/Nri/Ui/Button/V2.elm

```diff
@@ -595,17 +597,29 @@ buttonClass size width colorPalette =
             ]
 
 
-viewLabel : Maybe IconType -> String -> Html msg
+viewLabel : Maybe IconType -> String -> List (Html msg)
 viewLabel icn label =
     case icn of
         Nothing ->
-            Html.text label
+            renderMarkdown label
 
         Just iconType ->
-            span [ styles.class [ LabelIconAndText ] ]
-                [ decorativeIcon iconType
-                , Html.text label
-                ]
+            [ span [ styles.class [ LabelIconAndText ] ]
+                (decorativeIcon iconType
+                    :: renderMarkdown label
+                )
+            ]
+
+
+renderMarkdown : String -> List (Html msg)
```

Comment:
📝 this is used in https://github.com/NoRedInk/elm-formatted-text/blob/master/src/FormattedText/Markdown.elm#L62 as well

Reply:
👍